testsuite/features/step_definitions/command_steps.rb

```diff
@@ -24,9 +24,12 @@
 # rubocop:disable Metrics/BlockLength
 Then(/^it should be possible to reach the (.*)$/) do |resource|
```

Comment:
IMHO, after see the lenght of that block, I'd split it using a step definition for each.

Reply:
I hesitated a lot. Maybe it's better, yes.